boards/qemu-coreboot-fbwhiptail-tpm1-hotp/qemu-coreboot-fbwhiptail-tpm1-hotp.config

```diff
@@ -32,8 +32,8 @@ CONFIG_HOTPKEY=y
 
 #Uncomment only one of the following block
 #Required for graphical gui-init (FBWhiptail)
-CONFIG_CAIRO=y
-CONFIG_FBWHIPTAIL=y
+#CONFIG_CAIRO=y
+#CONFIG_FBWHIPTAIL=y
```

Comment:
Did you mean to commet this out?  I would like to keep an fbwhiptail test

Reply:
That was by error